rhizome/host/lib/vm_setup.rb

```diff
@@ -266,6 +277,12 @@ def generate_ip4_filter_rules(nics)
     end.join("\n")
   end
 
+  def generate_private_ip4_list(nics)
+    nics.map do |_, net4, _, _|
```

Comment:
I'm liking this quad-tuple less and less. But, it's already there. Can we pencil in a change to have it be a Struct or something?

Reply:
Yes, I feel the same, I will do it in a separate PR. I'll switch to a struct